src/store/plugins/errorHandler.js

```diff
@@ -0,0 +1,25 @@
+export default ({ dispatch }) => {
+  const errors = [];
+  let invoke = true;
+  const handlerBody = async (error) => {
+    if (error) errors.unshift(error);
+    if (invoke && errors.length !== 0) {
```

Comment:
```suggestion
    if (!invoke || errors.length === 0) return;
```
proposing to avoid extra nesting

Reply:
will keep that in mind